packages/flutter_tools/lib/src/version.dart

```diff
@@ -158,12 +187,12 @@ class FlutterVersion {
   /// Return the branch name.
   ///
   /// If [whitelistBranchName] is true and the branch is unknown,
-  /// the branch name will be returned as 'dev'.
+  /// the branch name will be returned as '[user-branch]'.
```

Comment:
Nit: surround '[user-branch]' in backticks so Dartdoc knows not to try to resolve it as a reference

Reply:
done